test/test_sunburst.py

```diff
@@ -0,0 +1,49 @@
+from pyecharts.charts import Sunburst
+
+
+def test_sunburst_base():
```

Comment:
💯 !

It will be better to use `patch`, to avoid saving things to harddisk.

https://github.com/pyecharts/pyecharts/blob/dev/test/test_bmap.py#L10